.github/workflows/build-docker-image.yml

```
@@ -0,0 +1,67 @@
+name: Build Docker Image
+
+on:
+  push:
+    branches:
+      - main
+      - develop
```

Let's use a consistent order for both lists:

```suggestion
      - develop
      - main
```